spec/client_spec.js

```diff
@@ -419,15 +419,15 @@ describe('Client', function() {
         it('resolves when the request succeeds', function(done) {
           triggerEvent(subject, 'request:' + requestsCount + '.done', response);
           promise.then(function() {
-            expect(doneHandler).to.have.been.calledWith(response.responseArgs[0]);
```

Comment:
I was aware and also definitely agree with @danielbreves.

Unfortunately how the data was passed, isn't how it works in Promises 😢 .

What are you thinking when you say 'different strategy'.